packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -93,9 +91,7 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
     })();
   }, [data, dataProvider, userId]);
 
-  /* @conditional-compile-remove(raise-hand) */
   let activePersona = '';
-  /* @conditional-compile-remove(raise-hand) */
   if (props.allowActiveBorder) {
     // Display a border for raised handed participants in participant list
     activePersona = mergeStyles({
```

Comment:
not this PR but looks like a bug, line 110 below this isn't doing anything, I think the props.styles should be merged here instead so its assigned to activePersona correctly